### ISSUE TITLE
Padlet

### ISSUE THREAD

**Issue by gerhardbeck** [junaid33/opensource.builders#422]
a Whiteboard

A true open source alternative is ideas of lerntools (www.codeberg.org/lerntools), testing possible under www.lerntools.org

**Comment by jcklpe** [comment]
@gerhardbeck : is padlet the open source or closed source software?

**Comment by jcklpe** [comment]
@gerhardbeck : Looking up Padlet it seems to be essentially the same thing as Miro, no?

**Comment by gerhardbeck** [comment]
Not really. Padlet is online partly really usefull for online collaboration and its only in some parts similar to a kanbanboard.
Lerntools is much easier to use as a kanbanboard and simpler in structure.
Just try it at www.lerntools.org

**Comment by jcklpe** [comment]
@gerhardbeck  : Miro is not a kanbanboard. It's a whiteboard app. Also I am sorry but I don't speak German so I am unable to try the application at the link provided.   
  
I'm labeling this invalid until more information can be provided.

**Comment by jcklpe** [comment]
@gerhardbeck  : Miro is not a kanbanboard. It's a whiteboard app. Also I am sorry but I don't speak German so I am unable to try the application at the link provided.   
  
I'm labeling this invalid until more information can be provided.

**Comment by gerhardbeck** [comment]
Padlet and it's open source alternative is not a whiteboard at all, it is more similar to a kanbanboard.

We have an english version, too, you need to change the settings into english:

![grafik](https://user-images.githubusercontent.com/41069802/130031523-ad66fd21-1e38-444a-8d0e-d3cc4a34c12c.png)

You can test it here: https://lerntools.org/app/#/ideas/test12

But best is to register as we have different modules

**Comment by jcklpe** [comment]
@gerhardbeck   : Apologies. I thought padlet was a whiteboard because your first post says: "a Whiteboard"  
  
Is lerntools comparable to something like Trello or Asana?

**Comment by gerhardbeck** [comment]
Sorry, my mistake. My first post is wrong. It's not a whiteboard.
It's more like trello, but much easier.
It's a way to collect and present ideas in an interactive way. Hard to explain. Please test our Alternative with the link provided above or check https://www.codeberg.org/lerntools

Padlet is found at https://padlet.com/

**Comment by jcklpe** [comment]
@gerhardbeck  : sounds good. I'm actually surprised we don't already have an issue for kanban stuff. (we have a ticket already for project management but I think kanban is specific enough that we can have a ticket just for it) I've edited the title to more widely reflect the closed source options, and I've updated your original post comment to have a description of the use case space. I will also provide some links to other kanban style open source applications in this thread.

**Comment by jcklpe** [comment]
Other open source kanban boards  
  
Wekan:  [https://wekan.github.io/](https://wekan.github.io/)  
  
Taiga: [https://www.taiga.io/](https://www.taiga.io/)  
  
Restyaboard: [https://restya.com/board/](https://restya.com/board/)  
  
Kanboard: [https://kanboard.org/](https://kanboard.org/)  
  
also Nextcloud has a pretty good kanban plugin called Decks: https://apps.nextcloud.com/apps/deck

**Comment by gerhardbeck** [comment]
As said before Padlet.and its OS Alternative Lerntools is not really a kanbanboard,just similar but with different intention.

But if you want to collect it all unser kanbanboard Please do not forget to mention https://Codeberg.org/lerntools

**Comment by mlunax** [comment]
https://www.openproject.org/

**Comment by sulsoltanoff** [comment]
Planka App https://planka.app/
GitHub: https://github.com/plankanban/planka

**Comment by krishnaraj** [comment]
[Kanri](https://www.kanriapp.com/)
